resources/js/Pages/Results.vue

```diff
@@ -281,6 +279,11 @@
 
                 this.clearSubmitConfirmation();
 
+                this.submitting = true;
```

Comment:
:P what is exactly the difference between having this here and a few lines below?

Reply:
no difference, it's the other lines that shouldn't have been before the axios call.